Automattic/vip
# Dev-env: Remove 8.0 & 7.4 from available PHP versions

## Description

To match what is available right now since we deprecated 7.4 & 8.0 off of VIP

## Pull request checklist


## New release checklist


## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-dev-env-create.js`
2. Expect to only see the three PHP version choices
```
 WordPress site title · VIP Dev
✔ Multisite · false
? PHP version to use …
❯ 8.2 (recommended)
  8.1
  8.3
```
